gcf_data_mapper/read.py

```diff
@@ -0,0 +1,36 @@
+import csv
+import json
+from typing import Any, Optional
+
+import click
+
+
+def read_csv(file_path: str) -> list[str]:
+    """Reads a csv file and returns a list of values"""
```

Comment:
Missing param and return in docstring, please see [here](https://github.com/climatepolicyradar/navigator-admin-backend/blob/2674536196424175743efe20d72af0cc8cd2942f/app/repository/document.py#L201) for example